src/docfx/data/docs/metadata.json

```diff
@@ -29,59 +29,6 @@
             "type": "boolean",
             "description": "Indicate whether the depot need to API Scan"
         },
-        "product": {
-            "type": ["array", "null"],
```

Comment:
only remove `htmlMetaName`. Other properties like `type` is still used for metadata validation

Reply:
Oh sorry. Fixed.